src/typeSafeCssModulesPlugin.ts

```diff
@@ -27,37 +27,49 @@ export const typeSafeCssModulesPlugin: PluginCreator<
             // Index the stylesheet so that we can lookup lines & columns
             const stylesheet = new Stylesheet(await fs.readFile(from, "utf8"));
 
-            const classes = [...Object.entries(exportTokens)].map(
-                ([localClassName, scopedClassName]) => ({
-                    localClassName,
-                    scopedClassName,
-                    variableName: toCamelCase(localClassName),
-                    location: stylesheet.findClass(localClassName),
-                }),
+            const variables = [...Object.entries(exportTokens)].map(
```

Comment:
Naming this `variables` threw me for a loop. Can we call it something else, maybe something like `cssModuleExports`?